src/server/apps/worldserver/worldserver.conf.dist

```diff
@@ -1283,11 +1283,11 @@ Visibility.GroupMode = 1
 #                     Min limit is max aggro radius (45) * Rate.Creature.Aggro
 #        Default:     100  - (Visibility.Distance.Continents)
 #                     170 - (Visibility.Distance.Instances)
-#                     533 - (Visibility.Distance.BGArenas)
+#                     250 - (Visibility.Distance.BGArenas)
```

Comment:
The number must also be updated right above in a comment

Reply:
done, nice catch!